.github/workflows/check_unexpected_changes.yaml

```diff
@@ -12,22 +12,22 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-name: License Check
-on: [push, pull_request]
+name: Check Unexpected Changes
+on: [pull_request]
 
 jobs:
   check:
-    runs-on: ubuntu-18.04
+    if: github.actor != 'copybara-service[bot]'
+    runs-on: ubuntu-20.04
     steps:
     - name: Check out repo
       uses: actions/checkout@v3
-    - name: Setup Go for https://github.com/google/addlicense
-      uses: actions/setup-go@v3
-      with:
-        go-version: '1.17'
-    - name: Get the license tool
-      run: go get -u github.com/google/addlicense@v1.0
-    - name: Run the license tool
-      run: addlicense .
-    - name: Ensure there is no diff
+
+    - name: Build the JSON Schema
+      uses: ./.github/workflows/build_jsonschema
+
+    - name: Check for schema changes
       run: git diff --exit-code .
+
+    - if: ${{ failure() }}
+      run: echo "::error title=The JSON Schema changed!::If this is unexpected, update your branch or check for an open PR with schema changes"
```

Comment:
Discussion/Future: The piece I'm unclear about is what happens if we update the gen.sh script? Does that require a maintainer override to merge? Or should changes to that script be ignored by this check also?